### ISSUE TITLE
nextcloud.com/blog/nextcloud-contributor-week-moving-digital/ has desing issues in Firefox and Chrome, please check

### ISSUE THREAD

**Issue by alitabak** [nextcloud/nextcloud.com#1259]
As I visited the following link: https://nextcloud.com/blog/nextcloud-contributor-week-moving-digital/ , I recognized the following issue in Firefox and Chrome. 
I think the idea is to have a fixed menu box, so even if someone scrolls down will be able to see it. 
However menu box from above follows below in a way as it is shown in the attached pictures. It seems that it does also happen in Chrome. FYI! 

![snip-nextcloud-I](https://user-images.githubusercontent.com/47600237/77123886-a6ad1180-6a41-11ea-8fab-3e374aba49e6.png)
![snip-nextcloud-II](https://user-images.githubusercontent.com/47600237/77123888-a745a800-6a41-11ea-88d1-68c95e971d0e.PNG)

**Comment by jospoortvliet** [comment]
Yeah this happens on various pages sometimes when scrolling up and down - the menu javascript has some issues. Sadly I didn't write it and don't know Javascript so I can't fix it :(
